specification/2.0/README.md

```diff
@@ -2470,14 +2470,14 @@ A set of parameter values that are used to define the metallic-roughness materia
    
 ### material.normalTexture
 
-A tangent space normal map. Each texel represents the XYZ components of a normal vector in tangent space. <TODO: Add more explaination on how XYZ is calculated>.
+A tangent space normal map. The texture contains RGB components in linear space. Each texel represents the XYZ components of a normal vector in tangent space. Red [0 to 255] maps to X [-1 to 1]. Green [0 to 255] maps to Y [-1 to 1]. Blue [128 to 255] maps to Z [0 to 1]. The normal vectors use OpenGL conventions where +X is right and +Y is up. +Z points toward the viewer.
 
 * **Type**: [`material.normalTextureInfo`](#reference-material.normalTextureInfo)
 * **Required**: No
 
 ### material.occlusionTexture
 
-The occlusion map is a greyscale texture, with white indicating areas that should receive full indirect lighting and black indicating no indirect lighting. <Todo: sRGB ? Color Space>.
+The occlusion map is a greyscale texture in sRGB color space, with white indicating areas that should receive full indirect lighting and black indicating no indirect lighting. If an alpha channel is present, it is ignored.
```

Comment:
Is this all because it's a grayscale texture and wanting to store the texture efficiently in memory? I don't know the details of the support for this on different platforms. From a data perspective, occlusion should be sRGB to avoid banding (just like color textures).